README.md

```diff
@@ -240,6 +217,9 @@ The `Update` redeemer allows the owner to change the asking price of their posit
 3. All new outputs to the address must contain the proper datum:
     - It must be an inline `SwapPrice` datum with a price > 0.
 
+![Update Swap](./images/Update.jpg)
```

Comment:
Prices are asked / offer which means the image should be offering ADA. Flip the ratio.

Reply:
Also rename the datums like the previous image.